python/lsst/daf/butler/core/s3utils.py

```
@@ -19,19 +19,30 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-__all__ = ("s3CheckFileExists", "bucketExists", "setAwsEnvCredentials",
+__all__ = ("getClient", "s3CheckFileExists", "bucketExists", "setAwsEnvCredentials",
            "unsetAwsEnvCredentials")
 
 import os
-
-try:
-    import boto3
-except ImportError:
-    boto3 = None
+import boto3
 
 from .location import ButlerURI, Location
 
 
+def getClient():
```

and this should raise ModuleNotFoundError if boto3 was not found.